indico/legacy/webinterface/tpls/ConfDisplayFrame.tpl

```diff
@@ -17,7 +17,7 @@ else:
             <div class="confTitle">
                 <h1>
                     <a href="${ displayURL }">
-                        <span class="conferencetitlelink" ${textColorStyle}>
+                        <span class="conference-title-link" ${textColorStyle}>
```

Comment:
I wonder how we will be able to make this sustainable in the future, as non-CERN users start using those stylesheets.